src/contents/contact.js

```diff
@@ -0,0 +1,42 @@
+const contactComponent = (() => {
+  const mainElement = document.createElement('main');
+  const contactContent = document.createElement('div');
+  const contactHeading = document.createElement('h1');
+  const contactText = document.createElement('div');
+  const contactImage = document.createElement('img');
+
+  function addElementsTextContent() {
```

Comment:
This is an example of the function keyword the former reviewer was referring to:
```suggestion
  const addElementsTextContent = () => {
```

Reply:
I got your point, but could you explain why it would be better to use this notation in this case?
I'm using arrow functions in places I think it would be easier to read the code (like in my eventListeners) but in this case, I think the code becomes more readable using 'function'. What do you think?

Reply:
I agree with you, however, this is a requirement to help us get used to using `es6` as you would mostly use it in a standard working environment. `es5` is already becoming a thing of the past (unless in a few cases), so it's much better to jump into the future using `es6+`. I hope that makes some sense.